docs/UserGuide.md

```diff
@@ -359,6 +364,7 @@ from the specified location entity in the database.
 Note: The prefixes vary across different modes. Refer to the examples below for more clarity.
 </div>
 
+Here are some examples of how the command works in each mode:
```

Comment:
This line can be removed since its similar to the one in the callout just above it

Reply:
Done